jest.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  verbose: true,
+  collectCoverage: true,
+  setupTestFrameworkScriptFile: '<rootDir>/src/tests/setupTests.js',
```

Comment:
This appears to be deprecated. Please find evidence to the contrary or an alternative implementation

Reply:
Yeah, investigating the right way to do the change this. Thanks